crates/revm/src/inspector.rs

```diff
@@ -44,24 +48,16 @@ pub trait Inspector<DB: Database> {
     fn call(
         &mut self,
         data: &mut EVMData<'_, DB>,
-        call: H160,
-        context: &CallContext,
-        transfer: &Transfer,
-        input: &Bytes,
-        gas_limit: u64,
+        inputs: &CallInputs,
         is_static: bool,
     ) -> (Return, Gas, Bytes);
```

Comment:
@onbjerg this is only change that is going to have a effect outside of revm. all those fields are now inside `CallInputs`/`CreateInputs` struct, so it will be relatively easy to upgrade.